packages/app/src/app/pages/Dashboard/Content/routes/Repositories/index.tsx

```diff
@@ -85,6 +92,30 @@ export const RepositoriesPage = () => {
         </Element>
       ) : null}
 
+      {!allowsPrivateRepos ? (
```

Comment:
If we're going to use it like this most often (meaning as inverted boolean) we might want to change it to something that explicitly says unauthorised. Just something to keep in mind.